airflow/providers/amazon/aws/sensors/sqs.py

```diff
@@ -48,13 +68,26 @@ def __init__(
         aws_conn_id: str = 'aws_default',
         max_messages: int = 5,
         wait_time_seconds: int = 1,
+        visibility_timeout: Optional[int] = None,
+        message_filtering: Optional[str] = None,
+        message_filtering_match_values: Optional[Any] = None,
+        message_filtering_config: Optional[Any] = None,
         **kwargs,
     ):
         super().__init__(**kwargs)
         self.sqs_queue = sqs_queue
         self.aws_conn_id = aws_conn_id
         self.max_messages = max_messages
         self.wait_time_seconds = wait_time_seconds
+        self.visibility_timeout = visibility_timeout
+
+        self.message_filtering = message_filtering
+        if message_filtering_match_values is not None:
+            if not isinstance(message_filtering_match_values, list):
+                message_filtering_match_values = [message_filtering_match_values]
+        self.message_filtering_match_values = message_filtering_match_values
```

Comment:
I wonder `message_filtering_match_values` should be a `set` instead, this can improve performance quite a bit since the check is in a pretty tight loop.

Reply:
Thanks for the feedback, I did not think of that. Fixed